process/incident_management.md

```diff
@@ -0,0 +1,263 @@
+
+The post mortem meeting should involve all internal parties who were involved in the recovery itself and should be scheduled publicly to allow anyone interested to join and listen in or contribute:
+* [mandatory] The incident commander
+* [mandatory] The incident technical lead
+* [optional] Every SRE team members involved (as much as possible given the timezone constraints)
+  * The more brain power we have in the meeting the more likely we are to find the best countermeasures to avoid reoccurrence
+* [optional] The Customer Support lead
+
+The incident technical lead is the owner of the post mortem review meeting and is making sure that it contains the following:
+* Executive summary with customer impact, severity, …
+* Root Cause Analysis
+* Timeline of events
+* List of action items and countermeasures with
+  * Owner
+  * Criticality
+  * Target date for delivery
+
+All action items and countermeasures should aim at (as much as possible):
+
+* Avoid recurrence
+* Fix root cause
+* Improve monitoring and alerting by following best practices
+  * do not alert on edge cases, alert should be based on symptoms not root causes
+* Increase resiliency (being mindful of cost)
+* Increase automated recovery (if it makes sense)
+* Decrease MTTR in case of recurrence (documentation for instance, …)
+* **Each action item must be tracked by a corresponding ticket in the SRE's backlog so that its implementation can be prioritized and tracked by the SRE leads**
+
+
+Questionnaire to be answered during the PMR:
+* How can we protect ourselves better from such an incident?
+* How can we ensure that the issue does not reoccur?
+* How can we recover quicker from such an incident?
+* How can we identify the issue quicker?
+
+## Detailed process steps
+This section aims at presenting the steps that need to be taken in case of a new incident.
+
+![flow chart](resources/im_flowchart.png)
+
+## Handoff
+Some major incidents will span shift handoff between regions. When this happens:
+
+1. Outgoing Incident Commander identifies an incoming Incident Commander
+2. Incoming Incident Commander identifies an incoming Incident Tech Lead, and any Supporting SRE engineers
+3. Outgoing incident personnel brief by incoming incident personnel at shift handoff. Be sure to include:
+    1. Technical status of the issue
+    2. List of open and eliminated hypothesis regarding root cause
+    3. Time and content of the most recent customer update
+    4. Communication channels in play, including: IM channel, tracking doc, conference bridge, and email thread
+    5. Key support personnel / stakeholders outside SRE engaged in the incident
+
+Note: Outgoing team members should disengage no later than 30 minutes after their region handoff time (handoff+30).
```

Comment:
I'm not sure how realistic this is

Reply:
Maybe this sentence, and the bullet points below could be converted to a Recommendation rather than a Note